database/upgrades/49-backfill-state-timestamp-bigint.sql

```diff
@@ -0,0 +1,4 @@
+-- v49: Convert first_expected_ts to BIGINT so that we can use zero-values.
+
+ALTER TABLE backfill_state
+ALTER COLUMN first_expected_ts TYPE BIGINT;
```

Comment:
This doesn't look very sqlite-compatible, but sqlite doesn't care about column types anyway, so just making sure the data is turned into unix timestamps is enough (or even just delete data if it's too much effort to convert)

https://github.com/mautrix/go/blob/master/util/dbutil/upgradetable.go#L84-L88 is the documentation on limiting parts of migrations to specific dialects

Reply:
Are the timestamps from WA in seconds or milliseconds after the epoch?

Reply:
I think WhatsApp uses seconds everywhere